build.proj

```diff
@@ -131,16 +130,11 @@
     <!-- Build the Microsoft.Azure.Build.Tasks project -->
     <Exec Command="dotnet publish $(RepoTools)BuildPackagesTask/Microsoft.Azure.Build.Tasks/Microsoft.Azure.Build.Tasks.csproj -c $(Configuration)" />
 
-    <!-- Get all of the files changed in the given pull request -->
-    <FilesChangedTask RepositoryOwner="Azure" RepositoryName="azure-powershell" PullRequestNumber="$(PullRequestNumber)"  OutputFile="$(RepoArtifacts)FilesChanged.txt">
```

Comment:
@wyunchi-ms & @Nickcandy , could you help me confirm if it's OK to remove this task?

Reply:
It's OK to remove it now.